parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -330,6 +338,30 @@ pub mod pallet {
 				.saturating_add(len_fee)
 				.saturating_add(T::PricingParameters::get().rewards.local)
 		}
+
+		/// Burn the amount of the fee embedded into the XCM for teleports
+		pub fn burn_fees(para_id: ParaId, fee: BalanceOf<T>) -> DispatchResult {
+			let dummy_context =
+				XcmContext { origin: None, message_id: Default::default(), topic: None };
```

Comment:
Should we add the actual message id and origin here since we have access to this?

Reply:
`XcmContext` is irrelevant in teleports checks so the dummy should be fine.